src/extends/room/construction.js

```diff
@@ -35,6 +35,25 @@ const skipStructures = [
   STRUCTURE_RAMPART,
   STRUCTURE_CONTAINER
 ]
+
+const orderStructures = [
+  STRUCTURE_SPAWN,
+  STRUCTURE_TOWER,
+  STRUCTURE_EXTENSION,
+  STRUCTURE_CONTAINER,
+  STRUCTURE_STORAGE,
+  STRUCTURE_LINK,
+  STRUCTURE_WALL,
+  STRUCTURE_EXTRACTOR,
+  STRUCTURE_TERMINAL,
+  STRUCTURE_LAB,
+  STRUCTURE_RAMPART,
+  STRUCTURE_OBSERVER,
+  STRUCTURE_POWER_SPAWN,
+  STRUCTURE_NUKER,
```

Comment:
I think the nuker should be built before the power spawn, so it's cooldown period can start sooner. Of course once power spawns come out I may have a different opinion.

Reply:
Never been to that stage so i'll trust you on your word haha. I'll update it in a bit.